src/components/atoms/Link/Link.tsx

```diff
@@ -1,15 +1,20 @@
-import React, { FC } from 'react';
+import React from 'react';
 import styled from 'styled-components';
-import { colors, IBaseColorSpec, INeutralColorSpec } from '../../../constants/colors';
+import { colors } from '../../../constants/colors';
 import Text from '../../atoms/Text/Text';
 
-export interface ILinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
-  color?: IBaseColorSpec['secondary'] | INeutralColorSpec['white'];
+export interface ILinkProps
+  extends React.AnchorHTMLAttributes<HTMLAnchorElement>,
+    React.RefAttributes<HTMLAnchorElement> {
```

Comment:
You can use this instead of those 2 extensions: `React.DetailedHTMLProps<React.AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>`
Actually I think we should be using always this. As this is the return type of `styled.a`

Reply:
I think we still need to reference `React.RefAttributes` in the type somehow, otherwise TS complains about `ref` below 😪